client/pages/BikeInfo.tsx

```diff
@@ -5,13 +5,15 @@ import {
     TouchableOpacity,
     SafeAreaView,
     StatusBar,
+    LogBox
 } from 'react-native';
 import Icon from 'react-native-vector-icons/FontAwesome';
 import FirebaseImg from '../components/FirebaseImg';
 // @ts-ignore
 import {Rating} from 'react-native-stock-star-rating';
 import {FirebaseImgProps} from '../types/types';
 import {addRide} from '../services/rides';
+import { makeBikeUnavailable } from "../services/bikes";
```

Comment:
I would just keep this as a simple patchBike or something more neutral like updateBikeStatus or even checkOutBike to match with the naming of checkOutBike

Reply:
Yeah maybe I'll just replace that with a generic _updateBike_ function for more portability.